src/scss/nl-design-system-community.scss

```diff
@@ -3,7 +3,6 @@
 @import '@utrecht/components/dist/form-field-description/css/index.css';
 @import '@utrecht/components/dist/form-label/css/index.css';
 @import '@utrecht/components/dist/img/css/index.css';
-@import '@utrecht/components/dist/link/css/index.css';
```

Comment:
promoted into our `_anchor.scss` file so it's included in our build artifact instead of only used for SB